27-cellular-connected-electronic-kiosk/README.md

```diff
@@ -0,0 +1,150 @@
+
+## microSD Card Setup
+
+Before installing the Raspberry Pi into the Smart Pi case, you need to install [Raspberry Pi OS](https://www.raspberrypi.com/software/) onto the microSD card by following the steps in [this official video](https://www.youtube.com/watch?v=ntaXWS8Lk34) from the Raspberry Pi Foundation. Alternatively, you can follow text-based documentation [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/2).
+
+With the microSD card ready to go, plug it into the Pi:
+
+![microSD card installation](images/pi-sd.png)
+
+## Hardware Setup
+
+### Notecard and Notecarrier
+
+To connect the Notecard and Pi Hat to the Pi, follow [the section "Connect Your Notecard and Notecarrier" from the Blues Quickstart guide](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-pi/#connect-your-notecard-and-notecarrier).
+
+### Smarti Pi Case and Display
+
+To put together the Pi, display, and case, you will follow [this documentation from Smarti Pi](https://cdn.shopify.com/s/files/1/0793/8029/files/touch_pro_assembly_instructions.pdf?v=1640377735), with a few additional steps to accommodate the Notecard hardware.
+
+At Step 7, make sure to install the Notecarrier Pi Hat with the Notecard onto the Pi's headers:
+
+![Pi Hat on Pi's headers](images/pi_hat.jpg)
+
+Everything should fit, even with the standoffs attached to the Pi. You will also need to use the micro USB cable to connect the Notecarrier's micro USB port to one of the Pi's USB ports:
+
+![Notecarrier USB connection](images/notecarrier_usb_connection.jpg)
+
+At this point, the assembly should look something like this:
+
+![Fully assembled](images/full_assembly.jpg)
+
+From here, you can continue with the rest of the steps in the Smarti Pi documentation.
+
+### Keyboard and Mouse
+
+Install the AAA batteries into the keyboard and mouse, and plug the USB dongle into a free USB port on the Pi. Note that this single dongle is used for both the keyboard and mouse.
+
+## Booting for the First Time
+
+Plug the USB-C power supply into the exposed USB-C port on the case and let the Pi boot up. Complete the setup steps as described in [this documentation](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/4).
+
+## Notecard Firmware
+
+The Notecard should use [firmware version 5.1.1](https://dev.blues.io/notecard/notecard-firmware-updates/#v5-1-1-april-5th-2023) or higher. The simplest way to update firmware is to do an [over-the-air update](https://dev.blues.io/notecard/notecard-firmware-updates/#ota-dfu-with-notehub).
+
+## Kiosk Software
+
+### Dependencies
+
+To run the kiosk software, you'll need at least Python 3.8. The default Python version that comes with the OS should be sufficient, but you can check the version at the command line with `python --version`. You will also need to install a couple of Python packages. From this directory, `27-cellular-connected-electronic-kiosk`, run `pip install -r requirements.txt` to install them.
```

Comment:
Adjust this directory to point to our newly created repo instead https://github.com/blues/accelerators-cellular-connected-electronic-kiosk